doc/api/tls.md

```diff
@@ -561,9 +561,8 @@ added: v0.11.4
 -->
 
 Returns an object representing the cipher name. The `version` key is a legacy
-field which always contains the value `'TLSv1/SSLv3'`.
-
-For example: `{ name: 'AES256-SHA', version: 'TLSv1/SSLv3' }`
+field which always contains the value `'TLSv1/SSLv3'`:
+`{ name: 'AES256-SHA', version: 'TLSv1/SSLv3' }`.
```

Comment:
This seems confusing to me. The former version was much clearer out of my perspective. The same applies to the part below.